docs/testing.md

```diff
@@ -0,0 +1,27 @@
+# How to test decidim engines
+
+## Requirements
+
+You need to install [chromedriver](https://sites.google.com/a/chromium.org/chromedriver/) to run feature specs.
```

Comment:
We should add a minimum Chrome version, right? I think the first one that accepts `--headless` is v59